rust/agents/relayer/src/relayer.rs

```diff
@@ -120,46 +121,71 @@ impl Relayer {
             outbox.chain_name(),
             inbox_contracts.inbox.chain_name(),
         );
+        info!(
+            name=%inbox_contracts.inbox,
+            db=?db,
+            outbox=?outbox,
+            metrics=?metrics,
+            gelato=?gelato_conf,
+            "running inbox message processor and submit worker"
+        );
+        let (snd, rcv) = tokio::sync::mpsc::channel(1000);
```

Comment:
Done. I have imported tokio::sync::mpsc, changed to an unbounded_channel to avoid the constant, and renamed snd/rcv to try to convey the purpose of the channels.